sites/public/src/LinkComponent.tsx

```diff
@@ -6,6 +6,7 @@ const LinkComponent = (props: PropsWithChildren<LinkProps>) => {
   const anchorProps = { ...props }
   delete anchorProps.href
 
+  /* eslint-disable jsx-a11y/anchor-has-content */
```

Comment:
This is the only rule that I disabled in both `LinkComponent` files in public and partners, which I think makes sense for our use here